src/averageAges.js

```diff
@@ -26,25 +12,6 @@ function calculateMenAverageAge(people, century) {
 
   return total / result.length;
 }
-// write code here
-// learn how to use array methods like .filter .map .some .every .find .reduce
-// avoid using loop and forEach
-// replace `if ()` statement with &&, || or ?:
-// without nesting
-/**
- * Implement calculateWomenAverageAge function
- *
- * Function returns average age of women in array. If `withChildren` is
- * specified then function calculates average age only for women with children
- *
- * Hint: To check if a woman has children you should find someone who mention
- * her as mother.
- *
- * @param {object[]} people
- * @param {boolean} withChildren - optional
- *
- * @return {number}
- */
```

Comment:
that's not the ordinary comment - it's [JS Doc](https://jsdoc.app/)
So in the future, you may leave JS Docs